lab-judy/server.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+//node modules
+
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+
+//npm modules
+const cowsay = require('cowsay');
+
+console.log(process.argv[0]);
```

Comment:
Please remove console logs prior to submission unless they're relevant to your program and not just for testing/debugging, such as console.errors on the server.